plugins/modules/k8s_info.py

```diff
@@ -42,10 +42,21 @@
     description: List of field selectors to use to filter results
     type: list
     elements: str
+  state:
+    description:
+    - Desired state of the object to gather information from.
+    - Only applicable if C(wait), C(wait_sleep), C(wait_condition), and C(wait_timeout) specified.
+    - If you are waiting for object to be created by Kubernetes, set this to C(present).
+    - If you are waiting for object to be deleted by Kubernetes, set this to C(absent).
+    - This parameter is different from regular Ansible state parameter.
```

Comment:
Is there a convention for this or does `state` still make the most sense?

Reply:
I'd prefer not to add it, but have `wait_state` if we really need it

Reply:
yeah, I think the `absent` behavior can actually be achieved like so:

```
k8s_info:
  api_version: v1
  kind: Namespace
  name: testing
register: result
until: not result.resources
```

so really no need for `state` here IMO

Reply:
I removed `state`.